book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.book = self
+  end
```

Comment:
- [ ] As requested by the previous code reviewer, which is also a requirement for this project, method for adding rentals should be present and the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association, something like so: 

```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```

Please kindly fix that.